docs/examples/model_repository/resnet_torchscript/Readme.md

```diff
@@ -0,0 +1,154 @@
+      %3209 : __torch__.torch.nn.modules.batchnorm.BatchNorm2d = prim::GetAttr[name="bn1"](%self.1)
+      %3203 : __torch__.torch.nn.modules.conv.Conv2d = prim::GetAttr[name="conv1"](%self.1)
+      %3854 : Tensor = prim::CallMethod[name="forward"](%3203, %input.1)
+      %3855 : Tensor = prim::CallMethod[name="forward"](%3209, %3854)
+      %3856 : Tensor = prim::CallMethod[name="forward"](%3210, %3855)
+      %3857 : Tensor = prim::CallMethod[name="forward"](%3211, %3856)
+      %3858 : Tensor = prim::CallMethod[name="forward"](%3299, %3857)
+      %3859 : Tensor = prim::CallMethod[name="forward"](%3413, %3858)
+      %3860 : Tensor = prim::CallMethod[name="forward"](%3579, %3859)
+      %3861 : Tensor = prim::CallMethod[name="forward"](%3667, %3860)
+      %3862 : Tensor = prim::CallMethod[name="forward"](%3668, %3861)
+      %3013 : int = prim::Constant[value=1]() # /root/miniconda3/lib/python3.6/site-packages/torchvision/models/resnet.py:214:0
+      %3014 : int = prim::Constant[value=-1]() # /root/miniconda3/lib/python3.6/site-packages/torchvision/models/resnet.py:214:0
+      %input : Float(1:2048, 2048:1) = aten::flatten(%3862, %3013, %3014) # /root/miniconda3/lib/python3.6/site-packages/torchvision/models/resnet.py:214:0
+      %3863 : Tensor = prim::CallMethod[name="forward"](%3671, %input)
+      return (%3863)
+    
+
+
+## Saving Model
+
+An TorchScript model is a single file that by default must be named `model.pt`.A minimal model repository for a single PyTorch model would look like:
+
+```bash
+<model-repository-path>/
+  <model-name>/
+    config.pbtxt
+    1/
+      model.pt
+```
+
+
+```python
+model_dir = "sample_models_dir/resnet_torchscript/1" # making directory as required by Triton 
+if not os.path.exists(model_dir):
+    os.makedirs(model_dir)
+```
+
+First convert your model from GPU to CPU and then save it. This is recommended because the tracer may witness tensor creation on a specific device, so casting an already-loaded model may have unexpected effects. Casting the model before saving it ensures that the tracer has the correct device information.
+
+
+```python
+torch.jit.save(traced_model, os.path.join(model_dir,"model.pt"))
+```
+
+## Config File Creation
+
+Each model input and output must specify a name, datatype, and shape.
+
+The name specified for an input or output tensor must match the name expected by the model. PyTorch Naming Convention: Due to the absence of names for inputs and outputs in a TorchScript model, the `“name”` attribute of both the inputs and outputs in the configuration must follow a specific naming convention i.e. `“<name>__<index>”`. Where `<name>` can be any string and `<index>` refers to the position of the corresponding input/output. This means if there are two inputs and two outputs they must be named as: `“INPUT__0”`, `“INPUT__1”` and `“OUTPUT__0”`, `“OUTPUT__1”` such that `“INPUT__0”` refers to first input and `INPUT__1` refers to the second input, etc.
```

Comment:
Point to the docs for the PyTorch config I/O naming convention here and simply state the convention i.e. `“<name>__<index>”`. One example is enough since it is already covered and explained in details in the PyTorch section of the model config docs.